### DISCUSSION TITLE
How to turn off legend in CIE Diagram?

### DISCUSSION THREAD

**Discussion by hminle** [colour-science/colour#754] [Q&A] [answered]
Hi Colour Team,

In the CIE Diagram, how should I turn off the legend?
I use this function `colour.plotting.plot_RGB_colourspaces_in_chromaticity_diagram_CIE1931`
But I got a very long legend, so I want either turn it off or change the legend?
![image](https://user-images.githubusercontent.com/16201681/99151021-48830d80-2666-11eb-84b2-728130cc98bc.png)

Thanks

**Comment by thomasmansencal** [comment, marked as answer]
Hi @hminle,

So the quick way to turn the legend off is to do that:

```python
colour.plotting.plot_RGB_colourspaces_in_chromaticity_diagram_CIE1931(['sRGB', 'ProPhoto RGB'], legend=False)
```

Alternatively, and probably better in your case:

```python
figure, axes = colour.plotting.plot_RGB_colourspaces_in_chromaticity_diagram_CIE1931(['sRGB', 'ProPhoto RGB'], standalone=False)
axes.legend().texts = ['sRGB (Adapted)', 'ProPhoto RGB']
colour.plotting.render()  # or plt.show()
```

Cheers,

Thomas

**Comment by hminle** [comment]
Thank you a lot @KelSolaar
